doc/man3/EVP_RAND.pod

```diff
@@ -267,6 +267,12 @@ Returns the state of the random number generator.
 
 Returns the bit strength of the random number generator.
 
+=item "fips-indicator" (B<OSSL_RAND_PARAM_FIPS_APPROVED_INDICATOR>) <integer>
```

Comment:
I don't see `OSSL_RAND_PARAM_FIPS_APPROVED_INDICATOR ` anywhere in the code, and I don't see `OSSL_DRBG_PARAM_FIPS_APPROVED_INDICATOR` in the `.pod` files - is that intentional?

Reply:
Damn aliases.

These are defined in `util/perl/OpenSSL/paramnames.pm` albeit without the `OSSL_` prefix (which the script adds & I really wish I hadn't done that way and will eventually address).

I'll change things to be more consistent.

Reply:
Done I hope.